dev/Expander/Expander.xaml

```diff
@@ -133,30 +131,28 @@
                             BorderBrush="{TemplateBinding BorderBrush}"
                             BorderThickness="{TemplateBinding BorderThickness}"
                             MinHeight="{TemplateBinding MinHeight}"
-                            MinWidth="{TemplateBinding MinWidth}"
-                            MaxWidth="{TemplateBinding MaxWidth}"
                             contract7Present:CornerRadius="{TemplateBinding CornerRadius}"
                             IsEnabled="{TemplateBinding IsEnabled}"
                             Padding="{TemplateBinding Padding}"
                             Style="{StaticResource ExpanderHeaderDownStyle}"
                             Content="{TemplateBinding Header}"
                             ContentTemplate="{TemplateBinding HeaderTemplate}"
                             ContentTemplateSelector="{TemplateBinding HeaderTemplateSelector}"
-                            HorizontalAlignment="{TemplateBinding HorizontalAlignment}"
+                            HorizontalAlignment="Stretch"
                             IsChecked="{Binding Path=IsExpanded, Mode=TwoWay, RelativeSource={RelativeSource TemplatedParent}}" />
                         <!-- The clip is a composition clip applied in code -->
                         <Border x:Name="ExpanderContentClip" Grid.Row="1">
                             <Border
-                                x:Name="ExpanderContent"                         
+                                x:Name="ExpanderContent"
                                 Visibility="Collapsed"
                                 Background="{ThemeResource ExpanderDropDownBackground}"
```

Comment:
Should this really be binding to the "ExpanderDropDownBackground" ? The name is a bit confusing. It should be named something like "ExpanderContentBackground". Also note that WPF applies the Background to the entire control. In this case setting Background would apply to the layout root Grid.

(WPF Example)
![image](https://user-images.githubusercontent.com/17993847/109709916-0aedb600-7b6b-11eb-910b-ec3652c94a88.png)

Reply:
I believe naming (DropDown/Content) needs to pass API reviews (?) It's an entirely new control that was really bare before this visual update which added all sorts of things, including animations and names and what not.

Reply:
We can't apply background to the whole thing. Not by design. So header and dropdown/content are separate. Since content can be custom, and literally anything can go into content presenter, the default style I'd expect to go onto header itself.

Reply:
(Which we style, as opposed to unknown content)

Reply:
I'll rename to ExpanderContentBackground then since it hasn't gone through API reviews yet.

Concerning whether the background should apply to the header/content/entire-control: I agree WPF isn't optimal here and this is a strange case. Therefore, it should apply to the ExpanderHeader toggle button or the ExpanderContent border. Your points make some sense for why it should apply to the header. Additionally, the header is the only part of the control allways visible (even when collapsed) so an argument can be made it's only logical to apply to the header.

However, Padding is what throws a wrench into this. The Padding property really only mades sense around the actual content of the control -- the ExpanderContent/ContentPresenter. This means the Padding property really needs to bind there instead of to the header as it is now. Doing this however then forces the background to also apply to the content area. We dare not have Padding apply to content and Background apply to header -- that inconsistency would be extremely counterintuitive for developers.

It would make the most sense honestly to add separate properties for HeaderBackground and perhaps a few others.

Reply:
I dont understand why padding on toggle button is an issue. The actual toggle button control does have padding. I still think that's where control padding should belong. And yes, we're styling the default/always visible part, which is the header.

Reply:
And no, I don't think padding should bind to content. Content is independent and can be styled in absolutely any way depending of what we feed into the content presenter.

Reply:
Padding, conceptually, goes around the content https://docs.microsoft.com/en-us/windows/uwp/design/layout/alignment-margin-padding#margin-and-padding. 

The content is not in the toggle button, it is within the content present of the content border. It won't make sense to the majority of developers if you make Padding apply to the header.

Reply:
The header/toggle button also has content. The expander in default state is just the header, which has content, which has padding. Since content/drop down can be styled in absolutely any other independent way, I don't see why you'd want padding to apply to the dropdown, now this is inconsistent. On overall control, which is header view in rest state, there's padding on toggle button + chevron. So I still don't understand the issue.

Reply:
Also, that's why DropDown might be a better name than 'content'. It clearly explains that the area we are talking about is the one sliding out of the header. Both header and drop down have content.

Reply:
> The header/toggle button also has content. The expander in default state is just the header, which has content, which has padding. 

I see your point but this is an incorrect interpretation. You are confusing elements in the visual tree (the header) with a logical tree concept. The control, the actual Expander, only has ONE content. 

```xaml
<controls:Expander IsEnabled="False" AutomationProperties.Name="ExpanderNotEnabled" ExpandDirection="Down" IsExpanded="False" Margin="12">
  <controls:Expander.Header>
    <TextBlock Margin="0,0,0,1">This expander is not enabled</TextBlock>
  </controls:Expander.Header>
  <TextBlock>This is the real content of the control and only ever goes one place: the ContentPresenter.</TextBlock>
</controls:Expander>
```

WinUI will place that Content within the ContentPresenter which is never moved to the header even when collapsed. Content of the high-level control is only ever one place. The ToggleButton header content is different from the logical content of the control itself. Conceptually these are two different things.

Reply:
> Also, that's why DropDown might be a better name than 'content'. It clearly explains that the area we are talking about is the one sliding out of the header. Both header and drop down have content.

"Content" is the term used throughout the control (ContentExpander for example) and it makes more sense to me that this term is consistent (assuming the logical content meaning). Some of the existing names like `ExpanderDropdownDownBorderThickness` were a bit "off" sounding as well. However, if it comes down to it I can revert the naming changes. It doesn't have much overall significance.

Reply:
Content seems like a better name. DropDown only makes sense when expanding down.

Reply:
Names retain 'Content' terminology and relevant control properties now bind to the content area. Marking resolved.